### PR TITLE
Enable selecting the sgen major mode

### PR DESCRIPTION
**Pull request by BrzVlad**

### CODE REVIEW

File: tools/mmp/driver.cs [hunk 1]
```diff
@@ -1080,6 +1080,11 @@ static string GenerateMain ()
 				if (App.EnableDebug)
 					sw.WriteLine ("\txamarin_debug_mode = TRUE;");
 
+				if (App.EnableSGenConc)
```
Comment (rolfbjarne) [inline tools/mmp/driver.cs]: Does desktop AOT-compiled code have to be compiled with the same MONO_GC_PARAMS as is used at runtime?

Comment (BrzVlad) [inline tools/mmp/driver.cs]: No. It is safe to switch between sgen modes at runtime.

**Comment by donblas** [comment]
Neither test failure is your fault.

**Comment by donblas** [comment]
Woops. Wrong account (personal). Fixing

**Comment by chamons** [comment]
Neither test failure is your fault. Ignore the other account approving, chrome had me logged in as personal for some crazy reason. 👍

**Comment by chamons** [comment]
https://bugzilla.xamarin.com/show_bug.cgi?id=51717 for the XM test failure.

**Comment by chamons** [comment]
@rolfbjarne 👍  ?

**Pull request merged**